sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -1179,4 +1179,13 @@ class SparkSqlAstBuilder extends AstBuilder {
       }
     }
   }
+
+  override def visitShowProcedures(ctx: ShowProceduresContext): LogicalPlan = withOrigin(ctx) {
+    val ns = if (ctx.identifierReference() != null) {
```

Comment:
Minor: Why `()` at the end if not used on the line below? It seems like a simple getter where we can omit the parentheses.